lib/commands/context.js

```diff
@@ -109,9 +108,8 @@ commands.getWindowHandles = async function () {
     throw new errors.NotImplementedError();
   }
 
-  let pageArray = await this.listWebFrames();
-  this.windowHandleCache = _.map(pageArray, this.massagePage);
-  let idArray = _.map(this.windowHandleCache, 'id');
+  this.windowHandleCache = await this.listWebFrames(false);
+  const idArray = _.map(this.windowHandleCache, 'id');
```

Comment:
I cannot understand what this _.map is doing

Reply:
It's pulling the `id` property out of the array.

Reply:
It uses the `_.property` shorthand. https://lodash.com/docs/4.17.10#property

Reply:
it would be useful to have some docs on how `listWebFrames` result looks like

Reply:
I'd also rename this variable to `frameIds` or something like that